modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
+= Modifying the retention time and size for Prometheus metrics data
+
+By default, Prometheus automatically retains metrics data for 15 days.
+You can modify the retention time to change how soon data is deleted by specifying a time value in the `retention` field.
+You can also configure the maximum amount of disk space used by the retained metrics data by specifying a size value in the `retentionSize` field.
+If this size limit is reached, the oldest data is deleted first until the disk space used by the data is again below the maximum.
```

Comment:
Maybe rewrite to avoid passive voice?
```suggestion
If the data exceeds the size limit, Prometheus deletes the oldest data until the data no longer exceeds the limit.
```

Reply:
or 
```suggestion
If the data reaches this size limit, the oldest data is deleted first until the disk space used is again below the maximum.
```

Reply:
I edited to use the 2nd suggestion.